django/contrib/admin/checks.py

```diff
@@ -28,6 +29,18 @@ def _issubclass(cls, classinfo):
         return False
 
 
+def _module_exists(dotted_path, paths):
```

Comment:
I think the `_module_exists` and argument name could be improved. What about

```suggestion
def _contains_subclass(subclass_path, candidate_paths):
```